lib/rubocop/cop/lint/literal_as_condition.rb

```diff
@@ -18,12 +18,12 @@ module Lint
       #   end
       #
       #   # bad
-      #   if some_var && true
+      #   if true && some_var
```

Comment:
I was referring to this snippet - I was thinking it'd be good to have some note explaining why care only for `true` as the first operand.

Reply:
I've added a note.